### PR TITLE
[205] Support icon-nickname-switch

### LINKED ISSUE #205: Support icon-nickname-switch
News: https://developers.line.biz/en/news/2020/03/17/icon-nickname-switch/
Doc: https://developers.line.biz/en/docs/messaging-api/icon-nickname-switch/

### PR DESCRIPTION
**Pull request by cm-nakamura-yuki**
# Issue
fix #205 

# Change
- Add type of sender object

### CODE REVIEW

File: lib/types.ts [hunk 1]
```diff
@@ -497,6 +497,7 @@ export type MessageCommon = {
    * the quickReply property of the last message object is displayed.
    */
   quickReply?: QuickReply;
+  sender?: Sender;
```
Comment (xingoxu) [inline lib/types.ts]: Would you please add following comments above this property?
```
/**
 * [Change icon and display name](https://developers.line.biz/en/docs/messaging-api/icon-nickname-switch/)
 * 
 * When sending a message from the LINE Official Account, you can specify the `sender.name` and the `sender.iconUrl` properties in [Message objects](https://developers.line.biz/en/reference/messaging-api/#message-objects).
 */
```

File: lib/types.ts [hunk 2]
```diff
@@ -1744,6 +1745,22 @@ export type QuickReplyItem = {
...
+   * - Max: 20 characters
+   * - Certain words such as LINE may not be used.
+   */
+  name?: string;
+  /**
+   * Icon image URL (Max: 1000 characters)
+   *
+   * - **HTTPS**
+   */
+  iconUrl?: string;
```
Comment (xingoxu) [inline lib/types.ts]: Would you please change the comments to the following one? I will be very appreciated.
```
/**
* URL of the image to display as an icon when sending a message
*
* - Max character limit: 1000
* - URL scheme: https
*/
```

File: lib/types.ts [hunk 3]
```diff
@@ -1744,6 +1745,22 @@ export type QuickReplyItem = {
   action: Action;
 };
 
+export type Sender = {
+  /**
+   * Display name
+   *
+   * - Max: 20 characters
+   * - Certain words such as LINE may not be used.
```
Comment (xingoxu) [inline lib/types.ts]: Please quote 'LINE' to  \`LINE\`, so it can be `LINE`.

**Comment by xingoxu** [comment]
Thank you!

**Comment by cm-nakamura-yuki** [comment]
@xingoxu Thank you for reviewing.

**Pull request merged**